R/repo.R

```diff
@@ -63,15 +63,17 @@ package_repo <- function(desc, meta) {
     return(meta[["repo"]])
   }
 
-  # Otherwise try and guess from BugReports + URLs
+  # Otherwise try and guess from URLs + BugReports
   urls <- c(
-    desc$get_field("BugReports", default = character()),
-    desc$get_urls()
+    desc$get_urls(),
+    sub("/issues/?", "/", desc$get_field("BugReports", default = character()))
   )
 
   gh_links <- grep("^https?://git(hub|lab)\\..+/", urls, value = TRUE)
   if (length(gh_links) > 0) {
-    return(repo_meta_gh_like(gh_links[[1]], meta[["repo"]][["branch"]]))
+    return(repo_meta_gh_like(gh_links[[1]],
+                             meta[["repo"]][["branch"]],
+                             grepl("^https?://gitlab\\.", gh_links[[1]])))
```

Comment:
I think this logic would make more sense inside of `repo_meta_gh_like()`

Reply:
You're absolutely right, done.